### ISSUE TITLE
Items on BlockGroup still not gets serialized

### ISSUE THREAD

**Issue by stefanolsenn** [PiranhaCMS/piranha.core#558]
ref: #543 

Same approach, but now the items gets serialized as individual blocks instead of getting added to items array of the blockgroup.
```
"blocks":[
   {
      "body":{
         "value":"<p>&nbsp;afsdfsdfsdfsdfsdfsdf<br></p>"
      },
      "type":"Piranha.Extend.Blocks.HtmlBlock"
   },
   {
      "items":[],
      "type":"DPV.Web.Models.Blocks.LinkBlock"
   },
   {
      "hyperLink":{
         "value":"jkliiio"
      },
      "icon":{
         "id":"a43926f7-c7c9-45da-987c-426bffe42459",
         "hasValue":true
      },
      "picture":{
         "id":"a43926f7-c7c9-45da-987c-426bffe42459",
         "hasValue":true
      },
      "subTitle":{
         "value":"jkhjkukuik"
      },
      "title":{
         "value":"jkhjkjkjkjkjkjkjkjk"
      },
      "type":"DPV.Web.Models.Helpers.Link"
   }
]
```
I'm using version 6.0

**Comment by tidyui** [comment]
This is my bad. This was identified and fixed during testing, but unfortunately I seem to have applied to fix to the wrong branch.

**Comment by tidyui** [comment]
Fixed as of commit 96a41478619e8ee5dcf2ecbf17e08a0638a2d254

**Comment by tidyui** [comment]
Deployed and available on NuGet as of version `6.0.1`
